src/CacheClient.cpp

```diff
@@ -198,7 +198,10 @@ void GetCachingBehaviourSettingsFromEnvironment(bool* attemptReads, bool* attemp
             *attemptWrites = true;
             continue;
         }
-        Croak("The cache behaviour string provided: %s contains a character that is not R or W", behaviour);
+        if (c == '_')
```

Comment:
@lucasmeijer I remeber you telling me about this but I can't remember what the reason was. Something about hard to detect "<no value>".

Reply:
Yeah, it's impossible to set the cache behavior to "off" through env variables if we only allow R and W characters, because you cannot set an empty string as an env variable.

Reply:
Oh, windows. TIL. I think we should use something more descriptive though, how about one of the following?
* `off`
* `disabled`
* `false`

Reply:
agree, will adjust